moveit_ros/hybrid_planning/local_planner/local_constraint_solver_plugins/src/forward_trajectory.cpp

```diff
@@ -108,7 +114,12 @@ ForwardTrajectory::solve(const robot_trajectory::RobotTrajectory& local_trajecto
     {
       planning_scene_monitor::LockedPlanningSceneRO locked_planning_scene(planning_scene_monitor_);
       current_state = std::make_shared<moveit::core::RobotState>(locked_planning_scene->getCurrentState());
-      is_path_valid = locked_planning_scene->isPathValid(local_trajectory, local_trajectory.getGroupName(), false);
+      current_state->updateCollisionBodyTransforms();
+      collision_result_.clear();
+      collision_request_.group_name = local_trajectory.getGroupName();
+      locked_planning_scene->getCollisionEnv()->checkRobotCollision(collision_request_, collision_result_,
```

Comment:
So instead of checking the local trajectory, we are only checking the current state now?

Reply:
To be honest, this PR is over a year old. It seems there was a bug with `isPathValid()` but I don't remember clearly any longer. I guess I'll revert this change since it would be nice to check the entire path.

Reply:
Ah, but the downside of `isPathValid()` is that there's no distance checking.

Reply:
Do you mean we don't get the distance to collision objects? Does this matter since we only care about a binary decision of whether the local trajectory is collision-free or not?

Reply:
I reverted this change since we don't currently have a good way to test it.